types/trusted-types/index.d.ts

```diff
@@ -29,34 +23,32 @@ declare global {
         private brand: true; // To prevent structural typing.
     }
 
-    class TrustedURL {
-        private constructor(); // To prevent instantiting with 'new'.
-        private brand: true; // To prevent structural typing.
-    }
-
-    interface TrustedTypePolicy {
-        readonly name: string;
-        createHTML(input: string): TrustedHTML;
-        createScript(input: string): TrustedScript;
-        createScriptURL(input: string): TrustedScriptURL;
-        createURL(input: string): TrustedURL;
-    }
-
     interface TrustedTypePolicyFactory {
         createPolicy<Keys extends keyof TrustedTypePolicyOptions>(
```

Comment:
we can actually also preserve the type of the `create*` functions to make this more typed:

```typescript
type TypedPolicy<Options extends TrustedTypePolicyOptions> = {
  name: string,
  createHTML(...args: Parameters<NonNullable<Options['createHTML']>>): TrustedHTML;
  createScript(...args: Parameters<NonNullable<Options['createScript']>>):  TrustedScript;
  createScriptURL(...args: Parameters<NonNullable<Options['createScriptURL']>>): TrustedScriptURL;
}

interface TrustedTypePolicyFactory {
  createPolicy<Options extends TrustedTypePolicyOptions>(
    policyName: string,
    policyOptions?: Options,
  ): Pick<TypedPolicy<Options>, 'name'|(keyof Options&keyof TrustedTypePolicyOptions)>;
}
```